src/scintillaeditor.cpp

```diff
@@ -540,27 +571,16 @@ void ScintillaEditor::replaceSelectedText(const QString &newText)
     if ((qsci->selectedText() != newText)&&(qsci->hasSelectedText())) qsci->replaceSelectedText(newText);
 }
 
-void ScintillaEditor::replaceAll(const QString &findText, const QString &replaceText)
+void ScintillaEditor::replaceAll(const QString &findText, const QString &replaceText)//QScintilla bug no longer an issue 
```

Comment:
If you're uncertain I'd say put it back the way it was. I'm afraid of reintroducing an old bug on systems with an older version of QScintilla.

Unless you feel like installing QScintilla 2.7, reproduce the bug, and check that it's fixed in the new code (which I totally understand not being worth doing)..